packages/terra-demographics-banner/src/_sharedObjects.jsx

```diff
@@ -1,23 +1,46 @@
 import classNames from 'classnames/bind';
 import React from 'react';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
 import styles from './DemographicsBanner.module.scss';
 
 const cx = classNames.bind(styles);
 
 // eslint-disable-next-line react/prop-types
-const DemographicsBannerValue = ({ label, value }) => (
-  <span className={cx('value')}>
-    { label && <span className={cx('value-label')}>{`${label}:`}</span> }
-    <b>{value}</b>
-  </span>
-);
+const DemographicsBannerValue = ({ label, value, abbrTitle }) => {
+  let valueLabelContent;
+
+  if (abbrTitle) {
+    valueLabelContent = (
+      <span className={cx('value-label')}>
+        <abbr className={cx('abbreviation')} title={abbrTitle} aria-hidden="true">
```

Comment:
The translations and props added to this component indicate we are adding full text labels to the component. Are we changing the default translations that are being displayed or is abbreviation the correct term here?

Reply:
The displayed translations should be what's currently displayed, and the default ones should be what they are. For the abbreviation, we want to display the abbreviated text, and have the full text displayed when this is hovered over or read through a screen reader.

Reply:
Also, I think it's unnecessary to have screenshot behavior for the hover effect. That's a native browser implementation, and I think wdio should only test functionality terra provides on top of what browsers do.